Sources/SwiftInspectorVisitors/Tests/GenericRequirementVisitorSpec.swift

```diff
@@ -180,10 +180,8 @@ final class GenericRequirementVisitorSpec: QuickSpec {
       context("visiting a syntax tree involving an associatedtype that has one generic constraint") {
         it("finds the generic requirements") {
           let content = """
-            public protocol Test {
-              associatedtype SomeType: SomeProtocol where
-                Element: AnyObject
-            }
+            associatedtype SomeType: SomeProtocol where
```

Comment:
now that we are scoping this visitor to only care about where clauses and not visit children in member declarations, we need to focus this content a bit to only provide a bare-bones associated type with a where clause.

The changes below follow this theme.